### PR TITLE
Removing editor warnings

### PR DESCRIPTION
**Pull request by rebeksburnett**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,23 +5,25 @@ We recommend starting with [Getting started](vipps-getting-started.md).
...
-- [Vipps API servers](#vipps-api-servers)
-- [Vipps request servers](#vipps-request-servers)
-- [Vipps API Lifecycle](#vipps-api-lifecycle)
-- [Settlements](#settlements)
-- [Vipps design guidelines](#vipps-design-guidelines)
-- [Questions?](#questions)
-
-Document version: 2.6.4.
+- [Vipps Developers](#vipps-developers)
+  - [Table of contents](#table-of-contents)
```
Comment (cloveras) [inline README.md:9]: The page title and the ToC itself can be skipped from the ToC.

File: README.md [hunk 2]
```diff
@@ -69,15 +72,15 @@ See our [Plugins section](https://github.com/vippsas/vipps-plugins).
...
+You can also subscribe to updates for specific incidents. For example:
 
-* Test environment: https://vipps-test.statuspage.io
-* Production environment: https://vipps.statuspage.io
-* BankID production: https://www.bankid.no/status/
-* BankID preprod: https://bankidpreprod.statuspage.io
+- Test environment: <https://vipps-test.statuspage.io>
+- Production environment: <https://vipps.statuspage.io>
+- BankID production: <https://www.bankid.no/status/>
+- BankID preprod: <https://bankidpreprod.statuspage.io>
```
Comment (cloveras) [inline README.md:81]: Should all links be in angle brackets? 🧐

File: README.md (hunk 1 above)
Comment (rebeksburnett) [reply README.md:9]: I will update this for all these md files here. I have a plugin that keeps the links up-to-date, so need to add  " " at the end of the heading line.

File: README.md (hunk 2 above)
Comment (rebeksburnett) [reply README.md:81]: The markdown editor checker complained about them, so I was trying to get them out of the way so I'd be sure to see the more important errors. But we can also ignore that error check.

**Pull request closed without merging**